cloud_info_provider/providers/openstack.py

```diff
@@ -115,15 +116,16 @@ def __init__(self, opts):
         # Select 'public', 'private' or 'all' (default) templates.
         self.select_flavors = opts.select_flavors
 
-    def _rescope_project(self, project):
+    def _rescope_project(self, project, project_domain_name):
         '''Switch to new OS project whenever there is a change.
 
            It updates every OpenStack client used in case of new project.
         '''
         if (not self.project or project != self.project):
-            self.opts.os_project_id = project
+            self.opts.os_project_name = project
```

Comment:
project ids are not scoped to domains, so it was working fine before (probably the name "project" for the field was not the best). We also need to decide what to do with ONE as "project" was also used as name of the group. Maybe we should use middleware specific names and in that case I would suggest to use "project_name" + "domain_name" or just "project_id". Then inside the function use whatever combination is available.

Thoughts @gwarf ?

Reply:
So as said @enolfc, it's probably best to keep the way it's working (without introducing the need to specify the domain name),
But for the sake of clarity, it would help to change `self.project` to `self.project_id`.

@orviz, If you want you can make a (probably new) PR for this, otherwise we will handle it.
Does it sound OK with you?

Reply:
Let's move back the discussion to #117 to agree on what to do to clarify and make this more flexible.

Reply:
Perfect, thanks @orviz!
We will continue this on the next PR :)
Be sure to introduce the new `auth` entry in the share hierarchy, as shown in #117 .
Please let us know if you need any support.